.github/workflows/azure-deploy.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         creds: ${{ secrets.AZURE_CREDENTIALS }}
     - name: Run Ansible Playbooks
       run: |
-        ansible-galaxy collection install azure.azcollection --force
+        ansible-galaxy collection install azure.azcollection:1.19.0 --force
```

Comment:
Is there a reason we need to force a specific version? Shouldn't we always just grab the latest?

Reply:
The latest version (2.0.0) which got released 3 weeks ago is throwing errors when used in GitHub Actions. So used the previous version (1.19.0).
https://github.com/ansible-collections/azure/releases